### ISSUE TITLE
Crashes on first start

### ISSUE THREAD

**Issue by samvrlewis** [samvrlewis/ozbargain-notify#1]
After installing/updating, the app crashes. Closing and then relaunching seems to fix.

May only be a problem on Android N.

**Comment by samvrlewis** [comment]
```
01-17 17:03:29.605 29197 29197 E chromium: [ERROR:interface_registry.cc(104)] Failed to locate a binder for interface: autofill::mojom::PasswordManagerDriver
01-17 17:03:29.622 29197 29197 D CordovaWebViewImpl: onPageDidNavigate(file:///android_asset/www/index.html)
01-17 17:03:29.624  4870 13531 W InputMethodManagerService: Window already focused, ignoring focus gain of: com.android.internal.view.IInputMethodClient$Stub$Proxy@207bcf attribute=null, token = android.os.BinderProxy@7b5a3d
01-17 17:03:29.687 29197 29197 W cr_BindingManager: Cannot call determinedVisibility() - never saw a connection for the pid: 29197
01-17 17:03:30.324  4870  7861 I ActivityManager: Process android.process.acore (pid 28570) has died
01-17 17:03:30.324  4870  7861 D ActivityManager: cleanUpApplicationRecord -- 28570
01-17 17:03:30.626 29197 29197 D JsMessageQueue: Set native->JS mode to EvalBridgeMode
01-17 17:03:31.248 29197 29197 W cr_BindingManager: Cannot call determinedVisibility() - never saw a connection for the pid: 29197
01-17 17:03:31.260 29197 29197 D CordovaWebViewImpl: onPageFinished(file:///android_asset/www/index.html#/settings)
01-17 17:03:31.585 16704 16704 I MemoryTrimmer: Trimming objects from memory, since app is in the background.
01-17 17:03:32.064 29197 29248 I App     : WARNING: Back Button Default Behavior will be overridden.  The backbutton event will be fired!
01-17 17:03:32.082 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/controllers.js: Line 32 : android
01-17 17:03:32.083 29197 29197 I chromium: [INFO:CONSOLE(32)] "android", source: file:///android_asset/www/js/controllers.js (32)
01-17 17:03:32.087 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/controllers.js: Line 15 : [object Object]
01-17 17:03:32.087 29197 29197 I chromium: [INFO:CONSOLE(15)] "[object Object]", source: file:///android_asset/www/js/controllers.js (15)
01-17 17:03:32.094 29197 29248 V PushPlugin: execute: action=unsubscribe
01-17 17:03:32.098 29197 29249 D PushPlugin: Unsubscribing to topic: over_100
01-17 17:03:32.117 29197 29249 W InstanceID/Rpc: Found 10013
01-17 17:03:32.146 29197 29248 V PushPlugin: execute: action=init
01-17 17:03:32.147 29197 29251 V PushPlugin: execute: data=[{"android":{"senderID":"987757989249","sound":false,"vibrate":false,"icon":"notify"}}]
01-17 17:03:32.152 29197 29251 V PushPlugin: execute: jo={"senderID":"987757989249","sound":false,"vibrate":false,"icon":"notify"}
01-17 17:03:32.156 29197 29251 V PushPlugin: execute: senderID=987757989249
01-17 17:03:32.181  7680 24240 W GCM     : Mismatched messenger
01-17 17:03:32.182  7680 24240 W GCM     : Invalid parameter sender
01-17 17:03:32.183 29197 29251 E PushPlugin: execute: Got JSON Exception INVALID_PARAMETERS
01-17 17:03:32.184 29197 29251 D PushPlugin: no iconColor option
01-17 17:03:32.184 29197 29249 E PushPlugin: Failed to unsubscribe to topic: over_100
01-17 17:03:32.184 29197 29249 E PushPlugin: java.io.IOException: INVALID_PARAMETERS
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.zzc.zzb(Unknown Source)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.zzc.zza(Unknown Source)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.InstanceID.zzb(Unknown Source)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.google.android.gms.gcm.GcmPubSub.unsubscribe(Unknown Source)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin.unsubscribeFromTopic(PushPlugin.java:379)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin.access$800(PushPlugin.java:30)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin$7.run(PushPlugin.java:226)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
01-17 17:03:32.184 29197 29249 E PushPlugin: 	at java.lang.Thread.run(Thread.java:761)
01-17 17:03:32.191 29197 29248 V PushPlugin: execute: action=unsubscribe
01-17 17:03:32.191 29197 29249 D PushPlugin: Unsubscribing to topic: over_50
01-17 17:03:32.233 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/services.js: Line 14 : Error unsubbing
01-17 17:03:32.233 29197 29197 I chromium: [INFO:CONSOLE(14)] "Error unsubbing", source: file:///android_asset/www/js/services.js (14)
01-17 17:03:32.234 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/services.js: Line 15 : INVALID_PARAMETERS
01-17 17:03:32.234 29197 29197 I chromium: [INFO:CONSOLE(15)] "INVALID_PARAMETERS", source: file:///android_asset/www/js/services.js (15)
01-17 17:03:33.372  7680 24240 W GCM     : Invalid parameter sender
01-17 17:03:33.376 29197 29249 E PushPlugin: Failed to unsubscribe to topic: over_50
01-17 17:03:33.376 29197 29249 E PushPlugin: java.io.IOException: INVALID_PARAMETERS
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.zzc.zzb(Unknown Source)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.zzc.zza(Unknown Source)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.InstanceID.zzb(Unknown Source)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.google.android.gms.gcm.GcmPubSub.unsubscribe(Unknown Source)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin.unsubscribeFromTopic(PushPlugin.java:379)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin.access$800(PushPlugin.java:30)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin$7.run(PushPlugin.java:226)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
01-17 17:03:33.376 29197 29249 E PushPlugin: 	at java.lang.Thread.run(Thread.java:761)
01-17 17:03:33.382 29197 29248 V PushPlugin: execute: action=unsubscribe
01-17 17:03:33.382 29197 29249 D PushPlugin: Unsubscribing to topic: all_deals
01-17 17:03:33.382 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/services.js: Line 14 : Error unsubbing
01-17 17:03:33.383 29197 29197 I chromium: [INFO:CONSOLE(14)] "Error unsubbing", source: file:///android_asset/www/js/services.js (14)
01-17 17:03:33.384 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/services.js: Line 15 : INVALID_PARAMETERS
01-17 17:03:33.384 29197 29197 I chromium: [INFO:CONSOLE(15)] "INVALID_PARAMETERS", source: file:///android_asset/www/js/services.js (15)
01-17 17:03:33.405  7680 24240 W GCM     : Invalid parameter sender
01-17 17:03:33.410 29197 29249 E PushPlugin: Failed to unsubscribe to topic: all_deals
01-17 17:03:33.410 29197 29249 E PushPlugin: java.io.IOException: INVALID_PARAMETERS
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.zzc.zzb(Unknown Source)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.zzc.zza(Unknown Source)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.google.android.gms.iid.InstanceID.zzb(Unknown Source)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.google.android.gms.gcm.GcmPubSub.unsubscribe(Unknown Source)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin.unsubscribeFromTopic(PushPlugin.java:379)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin.access$800(PushPlugin.java:30)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at com.adobe.phonegap.push.PushPlugin$7.run(PushPlugin.java:226)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
01-17 17:03:33.410 29197 29249 E PushPlugin: 	at java.lang.Thread.run(Thread.java:761)
01-17 17:03:33.418 29197 29248 V PushPlugin: execute: action=subscribe
01-17 17:03:33.419 29197 29249 D PushPlugin: Subscribing to topic: all_deals
01-17 17:03:33.421 29197 29249 E AndroidRuntime: FATAL EXCEPTION: pool-1-thread-2
01-17 17:03:33.421 29197 29249 E AndroidRuntime: Process: me.samlewis.ozbargainnotify, PID: 29197
01-17 17:03:33.421 29197 29249 E AndroidRuntime: java.lang.IllegalArgumentException: Invalid appInstanceToken: 
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at com.google.android.gms.gcm.GcmPubSub.subscribe(Unknown Source)
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at com.adobe.phonegap.push.PushPlugin.subscribeToTopic(PushPlugin.java:349)
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at com.adobe.phonegap.push.PushPlugin.access$700(PushPlugin.java:30)
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at com.adobe.phonegap.push.PushPlugin$6.run(PushPlugin.java:211)
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
01-17 17:03:33.421 29197 29249 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:761)
01-17 17:03:33.424  4870  7041 W ActivityManager:   Force finishing activity me.samlewis.ozbargainnotify/.MainActivity
01-17 17:03:33.424 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/services.js: Line 14 : Error unsubbing
01-17 17:03:33.424 29197 29197 I chromium: [INFO:CONSOLE(14)] "Error unsubbing", source: file:///android_asset/www/js/services.js (14)
01-17 17:03:33.425 29197 29197 D SystemWebChromeClient: file:///android_asset/www/js/services.js: Line 15 : INVALID_PARAMETERS
01-17 17:03:33.425 29197 29197 I chromium: [INFO:CONSOLE(15)] "INVALID_PARAMETERS", source: file:///android_asset/www/js/services.js (15)
01-17 17:03:33.443 29197 29197 D CordovaActivity: Paused the activity.
```

**Comment by samvrlewis** [comment]
Fixed in 400136a
